backend/openapi.yaml

```diff
@@ -2378,6 +2378,36 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /api/v1/thirty-year-regulation/reports/regulation-letter:
```

Comment:
We discussed with the customer whether they wanted to have access to old regulation reports, and they said no (at least for now), so I didn't include it. It should be easy to add, so I think I'll add it just in case.

On the URL, see [this comment](https://github.com/City-of-Helsinki/hitas/pull/253#discussion_r1145800264).